flashpaper.go

```diff
@@ -101,7 +102,7 @@ func secretHandler(w http.ResponseWriter, r *http.Request) {
 		default:
 			{
 				http.NotFound(w, r)
-				fmt.Fprintf(w, "You are likely to be eaten by a grue")
+				fmt.Fprintf(w, "The requested paper has expired or viewed by someone else.")
```

Comment:
Missing a word here and on line 89: 
`"The requested paper has expired or has been viewed by someone else."`

Could we pull the text into a variable to DRY it up since we are using this in both places?